.github/workflows/main.yml

```
@@ -28,7 +28,9 @@ jobs:
     - name: Install dependencies
       run: apm ci
     - name: Run tests 👩🏾‍💻
-      run: npm run test
+      run: |
+        apm install atom-ide-base
+        npm run test
```

This change isn't really needed

```suggestion
      run: npm run test
```